core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -201,7 +202,12 @@ object ConsumerPerformance {
       .withRequiredArg
       .describedAs("count")
       .ofType(classOf[java.lang.Integer])
-      .defaultsTo(1)
+      .defaultsTo(1)    
+    val propsFileOpt = parser.accepts("consumer-properties", "External consumer properties file.")
```

Comment:
In `ConsoleConsumer` we use `consumer.config`, should we be using the same here for consistency?